addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -337,6 +337,7 @@ export class HtmlField extends Component {
             const t = document.createElement('T');
             t.setAttribute('t-out', dynamicPlaceholder);
             this.wysiwyg.odooEditor.execCommand('insert', t);
+            this.wysiwyg.odooEditor._pluginCall('sanitizeElement', [this.wysiwyg.odooEditor.editable]);
```

Comment:
> If that is the reason of the _pluginCall in the previous commit then the change in the previous commit should be in this commit instead. In any case, this is another private method call, obviously.
:+1: 

> Maybe the commit message could give more explanation about what the problem actually is ? Say if someone would be looking at reverting this commit for whatever reason, how could they test if the bug it is intended to solve comes back ?

As the commit message says : `It ensure the new t-out tags are properly flagged as inline elements.`
This is the reason we need the sanitation.
Do you want me to be more specific ?
I can add a technical explanation about how the sanitization affect this in the css result if you want.

Reply:
When I look at commit messages outside of reviews, it's usually because I'm blaming a line that I'd like to remove and so I want to understand why it's there in the first place. Basically, when reading a commit message, I expect to find the answer to this question: "What would break if I remove this line ?" Therefore, I expect the commit message to either explain in details why something is a big deal (why t-out that are not properly flagged as inline elements is a big deal ?) or forego the details but provide steps to reproduce a symptom of the underlying issue, or both.